python/lsst/ts/weatherstation/csc.py

```diff
@@ -144,6 +145,7 @@ async def begin_disable(self, id_data):
             self.fault()
 
         await super().begin_disable(id_data)
+        return SalRetCode.CMD_INPROGRESS
```

Comment:
See my comment above about how to send the ack

Reply:
Thanks for the clarification. I adjusted the code accordingly.